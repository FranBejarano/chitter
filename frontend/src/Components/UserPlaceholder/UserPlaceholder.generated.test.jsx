import { createMemoryHistory } from '@remix-run/router';
import React from "react";
import { MemoryRouter } from "react-router-dom";
import renderer from 'react-test-renderer';
import UserPlaceholder from "./UserPlaceholder";

jest.mock("axios");
jest.mock("react-router-dom");

const renderTree = tree => renderer.create(tree);
describe('<UserPlaceholder>', () => {
  it('should render component', () => {
    const history = createMemoryHistory();
    const route = `/users/find/`
    history.push(route);

    expect(renderTree(<MemoryRouter history={history}><UserPlaceholder
    /></MemoryRouter >).toJSON()).toMatchSnapshot();
  });

});