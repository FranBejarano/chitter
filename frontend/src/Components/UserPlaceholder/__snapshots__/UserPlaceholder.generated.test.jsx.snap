// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserPlaceholder> should render component 1`] = `null`;
